### PR TITLE
Fix/#13

### PR DESCRIPTION
**Pull request by Mi-Lan**
- Fixed these 3 issues #13, #12 and #14

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-import LandingPageFooter from './components/LandingPage/Footer';
-import LandingPageHeader from './components/LandingPage/Header';
 
-import FAQPage from 'components/FAQPage';
-import BrandAssetsPage from 'components/BrandAssetsPage';
-import CodebasePage from 'components/CodebasePage';
-import Manifesto from './components/Manifesto';
-import WorkerCompensation from './components/WorkerCompensation';
+import LandingPageFooter from './components/footer/Footer';
+import LandingPageNavBar from './components/navigation/NavigationBar';
```
Comment (KadenZipfel) [inline src/App.js:7]: I would remove 'LandingPage' from the names here, so just 'Footer' and 'Navbar'

File: src/containers/LandingPageContainer.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```
Comment (KadenZipfel) [inline src/containers/LandingPageContainer.js:1]: Is this component used anywhere? Seems like it can be deleted

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

File: src/containers/LandingPageContainer.js (hunk 2 above)
Comment (Mi-Lan) [reply src/containers/LandingPageContainer.js:1]: Good find!

**Comment by 0xVenky** [comment]
Did testing of this PR and all seems okay. There is an error in the console related to Navbar, but the functionality seems to work.
image

**Comment by Mi-Lan** [comment]
> Did testing of this PR and all seems okay. There is an error in the console related to Navbar, but the functionality seems to work.
> image

@0xVenky I fixed those errors in the next commit...seems that it's some weird error like styled components are not recognising as property...

**Review by KadenZipfel — APPROVED** [review]

**Comment by 0xVenky** [comment]
Tested and works perfect.

**Pull request merged**